Library/Homebrew/extend/ARGV.rb

```diff
@@ -25,7 +25,7 @@ def formulae
   def resolved_formulae
     require "formula"
     @resolved_formulae ||= (downcased_unique_named - casks).map do |name|
-      if name.include?("/")
+      if name.include?("/") || File.exist?(name)
```

Comment:
I'm not sure the added ` File.exist?(name)` makes sense. Isn't `name` just the argument given via the command line, i.e. typically something like `ack`? What I'm trying to say, doesn't this simply check for a file of the same name in the current working directory? What was the original intention of adding this?